.coveragerc

```diff
@@ -1,3 +1,7 @@
 [report]
-include =
```

Comment:
let me make a few tests; I think the right option is neither one, but [source](https://coverage.readthedocs.io/en/coverage-4.2/source.html).